lib/supplejack_common/sitemap/base.rb

```diff
@@ -14,6 +14,7 @@ class Base < SupplejackCommon::Base
 
     	class_attribute :_record_selector
     	class_attribute :_namespaces
+      class_attribute :_document
```

Comment:
align ?

Reply:
it is in real life, just looks weird here for some reason